Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -1805,4 +1805,25 @@ void WebDriverConnection::delete_cookies(Optional<StringView> const& name)
     }
 }
 
+// https://w3c.github.io/webdriver/#dfn-get-computed-role
```

Comment:
This line should be;
```c++
// 12.4.9 Get Computed Role, https://w3c.github.io/webdriver/#dfn-get-computed-role
```

I know this is a bit confusing, but the difference between this line and the commented spec links just above this function is that this is a numbered spec API vs something like `delete_cookies` being an "abstract operation".